LuaRules/Configs/commandsIDs.lua

```diff
@@ -0,0 +1,14 @@
+--  Proposed Command ID Ranges:
```

Comment:
Please, move this file to your modules folder (it is indeed used just there)

Reply:
I'm not sure if you read my previous reaction on this. Once I edited the lines, the comment is lost :)

Reply:
Yeah, i didn't, sorry! Would you repeat it to me?

Reply:
Ok, this time I will be a bit shorter:

1) this file is setting rules, setting global constants. Such things have to be in LuaRules (or gamedata), not in widgets/modules, because cmdIDs is something other gadgets/widgets would need to know/share.
2) you are right current version is linked only from new UI. Two reasons behind it: 
2a) Originally I planned adding scatter gadget (new command) and buildspeed gadget for factories (25%, 50%, 75%, 100% settings commands) to s44 but refactoring of those old things would be costly so I re-planned the delivery of those features for next iteration. Those gadgets would use the constants stored here.
2b) The file provides documentaiton: shows people suggested ranges of commands for any new definitions we may declare for any gadget
3) Based on 2a it would make sense to remove this file completely once those commands do not practically exist but I kept it here because: temporary cutting the dependent logic in UI is not trivial + if someone would pick those cmdIDs in next weeks, it would be another thing to solve while I will port the gadgets.

So - I cleaned the file so it does not contain all other junk commands as you suggested in  your last deleted comment and keep just the necessary minimum - two commands + docu - to use the easiest way to go forward

Reply:
Fair enough to me